### PR TITLE
[Translation] XLIFF Add `id` to meta data.

### PR DESCRIPTION
**Pull request by SpacePossum**
| Q | A |
| --- | --- |
| Branch? | master |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | maybe/no |
| Deprecations? | no |
| Tests pass? | yes |
| License | MIT |

I would like to get the `id` attribute value from parsed XLIFF v1.2 files in the meta data (of the catalogue).

### CODE REVIEW

File: src/Symfony/Component/Translation/Loader/XliffFileLoader.php [hunk 1]
```diff
@@ -232,7 +238,7 @@ private function getXmlErrors($internalErrors)
         $errors = array();
         foreach (libxml_get_errors() as $error) {
             $errors[] = sprintf('[%s %s] %s (in %s - line %d, column %d)',
-                LIBXML_ERR_WARNING == $error->level ? 'WARNING' : 'ERROR',
+                LIBXML_ERR_WARNING === $error->level ? 'WARNING' : 'ERROR',
```
Comment (javiereguiluz) [inline src/Symfony/Component/Translation/Loader/XliffFileLoader.php]: This change seems unrelated, right?

**Comment by javiereguiluz** [comment]
👍 I like this change.

File: src/Symfony/Component/Translation/Loader/XliffFileLoader.php (hunk 1 above)
Comment (SpacePossum) [inline src/Symfony/Component/Translation/Loader/XliffFileLoader.php]: indeed, I'll revert it :+1:

**Comment by fabpot** [comment]
Thank you @SpacePossum.
